add-on/chrome/modules/tabFrame.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+/* global sendAsyncMessage */
```

Comment:
Don't you need to add `addEventListener` to this statement here? If it's already part of .eslintrc, you might want to put this one in there as well.